server/render/index.js

```diff
@@ -23,18 +21,18 @@ function bumpStat( group, name ) {
 	}
 }
 
-export function render( element ) {
-	memoizedRenderToString.cache = markupCache;
-
-	if ( ! memoizedRenderToString.cache.has( JSON.stringify( element ) ) ) {
-		bumpStat( 'calypso-ssr', 'loggedout-design-cache-miss' );
-	}
-
+export function render( element, path, initialState ) {
```

Comment:
> Yeah, also I was thinking the API could be something like render.set( key, element ) and render.get( key )

I think better to put a comment before the async `renderThemeSheet()` than expose the caching from the render module.